front/src/lib/getLastTransactions.js

```diff
@@ -16,5 +16,18 @@ export default function getLastTransactions(transactions) {
 		return bDate - aDate;
 	});
 
-	return [transactions[0], transactions[1], transactions[2]];
+	const res = [];
+	let count = 0;
+	for (let el of transactions) {
```

Comment:
there must be built in array method that takes first N elements of array.
Please use that.

Reply:
done